improver_tests/cli/test_init.py

```diff
@@ -348,5 +348,13 @@ def test_mismatched_close_brackets(self):
             unbracket(['foo', ']', 'bar'])
 
 
+def test_import_cli():
+    """Test if `import improver.cli` pulls in heavy stuff like numpy."""
+    import subprocess
+    import sys
+    script = 'import improver.cli, sys; assert "numpy" not in sys.modules'
+    subprocess.run([sys.executable, '-c', script], check=True)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5220193)